ricky9667/HushKeyboard
# Create app home screen

## Description

This PR builds the home screen of the HushKeyboard app.

## How to verify

- Verify if the home screen shows the UI properly in light theme and dark theme.
- Verify if "Enable keyboard" and "Select input method" works expectedly.

## Screenshots / Videos

| Light theme | Dark theme |
| --- | --- |
| <img width=400 src="https://github.com/ricky9667/HushKeyboard/assets/55730003/8b0ff612-0142-48e7-acc2-43ca4af61a79" /> | <img width=400 src="https://github.com/ricky9667/HushKeyboard/assets/55730003/0d5b7d76-ad1f-4b00-924b-a1500295f976" /> |


https://github.com/ricky9667/HushKeyboard/assets/55730003/312438bd-5d78-4163-9ed1-f83794b9c91f
